### DISCUSSION TITLE
State of the project

### DISCUSSION THREAD

**Discussion by Nuttymoon** [spec-first/connexion#1735] [General]
Hi,

First of all, thanks for the awesome work done in this project! :clap: 

We are considering using Connexion as part of the backend of our product and I saw that the last commit on `main` is dated from July 1st. Is the project still active? There are very interesting features in `3.0` that we would love to use!

Cheers

**Comment by RobbeSneyders** [comment]
Hi @Nuttymoon,

Thanks for opening this discussion. This project is still active, but development has been a bit on a lower level the past months due to holidays of both @Ruwann and myself. In the meantime, we have continued to collect feedback on 3.0 and respond to issues. There's a couple of feedback points we want to address, but then we finally want to promote 3.0 from alpha to GA. We didn't want to do this before / during summer since we would have less time to support users.

**Reply by Nuttymoon** [reply]
Awesome! Thanks for the quick reply.
